react/modules/module06/README.md

```diff
@@ -63,18 +72,6 @@ The score might be less, if:
 
 ### Theory
 
-SSR:
- - [Rendering types](https://dexlock.com/blog/frontend-rendering-ssg-vs-ssr-vs-csr-vs-isr/)
- - [React.renderToPipeableStream](https://beta.reactjs.org/reference/react-dom/server/renderToPipeableStream)
- - [React Hydration](https://beta.reactjs.org/reference/react-dom/client/hydrateRoot#hydrating-an-entire-document)
- - [Redux SSR](https://redux.js.org/usage/server-rendering)
- - [React SSR with Express.js](https://dev.to/juhanakristian/basics-of-react-server-side-rendering-with-expressjs-phd)
-
-Cypress:
-- [What is e2e testing?](https://circleci.com/blog/what-is-end-to-end-testing/)
-- [How to write e2e tests](https://docs.cypress.io/guides/end-to-end-testing/writing-your-first-end-to-end-test)
-
-Extra:
- - [NEXT.js](https://nextjs.org/)
- - [Remix](https://remix.run/)
- - [Gatsby.js](https://www.gatsbyjs.com/why-gatsby)
\ No newline at end of file
+- [SSR vs SSG](https://vercel.com/blog/nextjs-server-side-rendering-vs-static-generation)
+- [Next.JS](https://nextjs.org/)
+- [Pages API](https://nextjs.org/docs/pages)
```

Comment:
May be it is worth including some documentation on supporting RTK with NextJS?
Specifically, how to integrate RTK Query with NextJS (e.g. https://webcodingcenter.com/redux/Server-Side-Rendering)

Reply:
Added link to official docs with repo example